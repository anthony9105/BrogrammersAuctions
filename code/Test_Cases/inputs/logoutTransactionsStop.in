login adminUser1                  
logout
bid racecar            currentSeller21    
bid racecar57          currentSeller21    