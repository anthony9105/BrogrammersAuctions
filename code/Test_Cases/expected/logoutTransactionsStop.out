Login successful
Logout successful
Error. Transactions are not allowed until you are logged in to an account
Error. Transactions are not allowed until you are logged in to an account